frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx

```diff
@@ -225,29 +226,45 @@ export function ProjectTree(): JSX.Element {
                     const oldPath = dragEvent.active.id as string
                     const folder = dragEvent.over?.id
 
+                    // Check if this is a multi-drag operation
+                    const isMultiDrag = dragEvent.active.data?.current?.multiDrag === true
+                    const draggedItemIds =
+                        isMultiDrag && dragEvent.active.data?.current?.ids
+                            ? (dragEvent.active.data.current.ids as string[])
+                            : [oldPath]
+
                     if (oldPath === folder) {
                         return false
                     }
 
+                    // Process all the dragged items
                     if (folder === '') {
-                        const oldSplit = splitPath(oldPath)
-                        const oldFile = oldSplit.pop()
-                        if (oldFile && oldSplit.length > 0) {
-                            moveItem(oldPath, joinPath([oldFile]))
-                        }
+                        // Handle dropping to root
+                        draggedItemIds.forEach((itemPath) => {
+                            const oldSplit = splitPath(itemPath)
+                            const oldFile = oldSplit.pop()
+                            if (oldFile && oldSplit.length > 0) {
+                                moveItem(itemPath, joinPath([oldFile]))
+                            }
+                        })
                     } else if (folder) {
+                        // Handle dropping to a folder
                         const item = viableItems.find((i) => i.path === folder)
                         if (!item || item.type === 'folder') {
-                            const oldSplit = splitPath(oldPath)
-                            const oldFile = oldSplit.pop()
-                            if (oldFile) {
-                                const newFile = joinPath([...splitPath(String(folder)), oldFile])
-                                if (newFile !== oldPath) {
-                                    moveItem(oldPath, newFile)
+                            draggedItemIds.forEach((itemPath) => {
+                                const oldSplit = splitPath(itemPath)
+                                const oldFile = oldSplit.pop()
+                                if (oldFile) {
+                                    const newFile = joinPath([...splitPath(String(folder)), oldFile])
+                                    if (newFile !== itemPath) {
+                                        moveItem(itemPath, newFile)
+                                    }
                                 }
-                            }
+                            })
                         }
```

Comment:
So [there's a `moveCheckedItems` action](https://github.com/PostHog/posthog/blob/master/frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx#L79-L89) that already does all of this, plus handles nested folders when both the folder and its items are selected. I'd suggest using it over making multiple `moveItem` calls.